BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -32,6 +32,7 @@ public struct StandardImageIdentifiers {
 
     // Icon size 24x24
     public struct Large {
+        public static let analyticsTrackers = "analyticsLarge"
```

Comment:
Please read our [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/Strings,-%60AccessibilityIdentifiers%60,-%60ImageIdentifiers%60#standardimageidentifiers) on how to add these images correctly.